web/source/keyboards/kmwkeyboards.ts

```diff
@@ -1547,15 +1582,6 @@ namespace com.keyman.keyboards {
         Pstub['KL'] = 'undefined';
       }
 
-      // If language code already defined (or not specified in stub), check to see if stub already registered
-      for(Lk=0; Lk < this.keyboardStubs.length; Lk++) {
-        if(this.keyboardStubs[Lk]['KI'] == Pstub['KI']) {
-          if(Pstub['KLC'] == '' || (this.keyboardStubs[Lk]['KLC'] == Pstub['KLC'])) {
-            return 1; // no need to register
-          }
-        }
-      }
```

Comment:
I can tell that this was transplanted to within `KeyboardInterface`.  Still... not exactly sure that wholesale-removing it is 100% correct.  Mostly because it feels really weird to only return `Promise.resolve(false)` and never a `Promise.resolve(true)`.

That said... I think it actually _is_ fine here, for this case, because this specific function _is_ only referenced by the `KeyboardInterface` API func.